.github/workflows/build-matrices.yaml

```diff
@@ -124,7 +124,11 @@ jobs:
         container: ${{ matrix.result.container_name }}
         prefix: ${{ matrix.result.command_prefix }}
         filename: ${{ steps.relative_path.outputs.relative_path }}
-      run: /bin/bash .github/scripts/build_simple.sh
+      run: |
+        /bin/bash .github/scripts/build_simple.sh
+        res=$?
+        df -h # In case of out of space, hopefully show *where*
+        exit $res
```

Comment:
It typically runs in strict so it will just fail as soon as it does and the workflow will stop. What you could do is change that, or try adding a block AFTER that does:

```
if: ${{ failure() }}
```
and then try to cat the spack failed log or something like that.

Reply:
All fixed, yay for strict mode not being strict in the middle of expression evaluation. 😁